syntax="proto2";
package adinfo;

message AdAccount {
    required uint64 ad_id = 1;
}

message AdInfo  {
    required AdAccount account = 1;
}
